core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -68,9 +68,10 @@ protected InodeDirectory getThis() {
    * Adds the given inode to the set of children.
    *
    * @param child the inode to add
+   * @return true if inode was added successfully, false otherwise
    */
-  public void addChild(Inode<?> child) {
-    mChildren.add(child);
+  public boolean addChild(Inode<?> child) {
```

Comment:
currently, it is not being used, but I want to use it in the future. Should I just keep it `void` and change it later?